Makefile

```diff
@@ -197,7 +197,7 @@ clean:
 
 .PHONY: test
 test: vendor $(ADDON_ASSET_FILE)  ## Run unit tests
-	@go test -tags "$(BUILD_TAGS)" -ldflags="$(VERSION_VARIABLES)" $(shell $(PACKAGES))
+	@go test -v -tags "$(BUILD_TAGS)" -ldflags="$(VERSION_VARIABLES)" $(shell $(PACKAGES))
```

Comment:
I am not sure why verbose flag is removed from the Makefile, this really nice to see each section of the test case and in case skip we can also get to know. I am adding back this one.

Reply:
:+1:

Reply:
👍 . I am not sure either how it got removed.